alcor/services/simulations/service.py

```diff
@@ -61,6 +103,9 @@ def save_parameters(*,
 def run_simulation(*,
                    parameters_values: Dict[str, NumericType],
                    model_type: int,
+                   geometry: str,
+                   cone_longitude: float,
+                   cone_latitude: float,
```

Comment:
```python
# I don't need these values in case of spherical geometry
# TODO: make them optional?
longitude = .0
latitude = .0
```
можно задать значения по умолчанию
```python
def run_simulation(*,
                   parameters_values: Dict[str, NumericType],
                   model_type: int,
                   geometry: str,
                   cone_longitude: float = 0.,
                   cone_latitude: float = 0.,
                   output_file_name: str) -> None:
```